### ISSUE TITLE
New release 1.30.0 exits immediately

### ISSUE THREAD

**Issue by alexeagle** [bazelbuild/rules_sass#127]
CI on this repo is red, and I observe the same by trying to upgrade to latest in a Bazel project.

This is the release including my recent rules_nodejs bazel changes, which makes it suspect. Maybe I broke it? However if I just apply this patch to revert the `sass.dart.js` version it works

```
diff --git a/sass/package.json b/sass/package.json
index 584a86c..1f54da3 100644
--- a/sass/package.json
+++ b/sass/package.json
@@ -1,6 +1,6 @@
 {
   "devDependencies": {
     "@bazel/worker": "2.3.1",
-    "sass": "1.30.0"
+    "sass": "1.29.0"
   }
 }
```

And reverting other parts of my recent change did not fix the problem.

**Comment by nex3** [comment]
It's possible this was caused by changing `main()` to return a JS `Promise` rather than a Dart `Future`. Are you running in worker mode or standard mode?

**Comment by alexeagle** [comment]
CI here is failing
https://buildkite.com/bazel/rules-sass/builds/866#290c8ddf-0aa4-4c1e-84db-b0d938b422e5
so you don't really need my involvement I think, but when I debugged this last week it was going down the single build path here  https://github.com/bazelbuild/rules_sass/blob/master/sass/sass_wrapper.js#L35

**Comment by BobobUnicorn** [comment]
Sorry about the delay; should be fixed with #128.

**Comment by alexeagle** [comment]
I guess #128 fixed it, I haven't seen this again.
